meredithisme/day2.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- Bootstrap CDN -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link rel="stylesheet" href="style.css">
+   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  <title>Day 2 Homework Bootstrap</title>
+</head>
+  <body>
+
+ <!--====== NAVIGATION =====-->
+
+    <nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <!-- Brand and toggle get grouped for better mobile display -->
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="infobar"></span>
+          <span class="infobar"></span>
+          <span class="infobar"></span>
+        </button>
+        <a class="navbar-brand" href="http://meredithisme.com">Meredith</a>
+      </div>
+
+    </div><!-- /.container-fluid -->
+  </nav>
+
+<!--====== END NAVIGATION =====-->
+
+
+<!--======= COLUMN LAYOUT =====-->
+
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 col-sm-6">
+          <h1>Homework Questions</h1>
+            <ul>
+              <li>Do RegExp find the matching word in a string? When is this utilized?</li>
+              <li>Do you use ternary operators often? And when to identify them</li>
```

Comment:
use them only for very simple operations like if something is true call functionA, else call functionB